src/main/java/hu/oe/nik/szfmv/automatedcar/model/World.java

```diff
@@ -1,44 +1,101 @@
 package hu.oe.nik.szfmv.automatedcar.model;
 
+import java.awt.*;
 import java.util.ArrayList;
 import java.util.List;
 
-public class World {
+/**
+ * World object containing the Model of the world
+ */
+public class World implements IWorld {
     private int width = 0;
     private int height = 0;
     private List<WorldObject> worldObjects = new ArrayList<>();
+    private int color;
 
-    public World(int width, int height) {
+    /**
+     * Initializes a new instance of the {@link World} class
+     * @param width Width of the {@link World}
+     * @param height Height of t he {@link World}
+     */
+    public World(int width, int height){
```

Comment:
kell nekünk ez a konstruktor?

Reply:
Igazából csak default konstruktora van szükségünk, a többit majd megcsinálja helyettünk a JAXB. Ahol majd kényelmes lesz a paraméterezhető konstruktor, az pl a Car osztály. Ezzel kapcsolatban kellene még konzultálni.

Reply:
A skeletonhoz való kompatibilitás miatt maradt benne, később nyugodtan kivehető lesz.

Reply:
A kezdőkódot (aka skeleton) át lehet írni. Talán 2 osztályban van komment, hogy nem (mindkettő a buszhoz kötődik), de még ott is elfogadjuk ha ez kellően indokolt.
A skeleton egyetlen funkciója, hogy segítsen elkezdeni a munkát. Már eleve azzal, hogy van egy maven projekt, ami valamit csinál, amihez csak hozzá kell nyúlni. Továbbá példát adjon rajzolásra, billentyűzet esemény kezelésre és legfőképpen a kommunikációs busz működésére.

Reply:
Szerintem nem része a #16-nak az World objektum létrehozásának átírása, majd a #17 az XML beolvasásnál átírja a megfelelő helyen.

Reply:
> Szerintem nem része a #16-nak az World objektum létrehozásának átírása, majd a #17 az XML beolvasásnál átírja a megfelelő helyen.

Egyetértek Tamással, zárnám is ezt a conversation-t.